src/components/ui/MultipleSelect/MultipleSelect.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import {Select} from 'antd'
+import {SelectProps} from 'antd/lib/select'
+import {WidgetField} from '../../../interfaces/widget'
+import {MultivalueSingleValue} from '../../../interfaces/data'
+
+type AdditionalAntdSelectProps = Partial<Omit<SelectProps, 'onChange' | 'value' | 'className' | 'disabled' | 'placeholder'>>
+
+export interface MultipleSelectProps extends AdditionalAntdSelectProps {
+    value: MultivalueSingleValue[],
+    values: Array<{ value: string }>,
+    onChange?: (value: MultivalueSingleValue[]) => void
+    // field common props
+    meta: WidgetField,
+    cursor: string,
+    widgetName: string,
+    className: string,
+    metaError: string,
+    disabled: boolean,
+    placeholder: string,
+    readOnly: boolean,
+    backgroundColor: string,
+    onDrillDown: () => void,
+}
+
+const MultipleSelect: React.FunctionComponent<MultipleSelectProps> = (props) => {
```

Comment:
I can't understand the difference with our regular `<Dictionary />` component, implementation looks almost identical aside of `multiple` option passed to the underlying antd `<Select />` and some missing features.

Wouldn't it be easier to make something like this:
```tsx
case FieldType.dictionary:
    resultField = <Dictionary
        {...commonProps}
        multiple={props.widgetFieldMeta.multiple}
        onChange={handleChange}
    />
```
We do support custom options for fields description in widget meta.

Reply:
@Dergash 
I thought about that. Here some reasons why decided to implement an another component:
- New component has simple implementation.
- As `Dictionary` component can get single value it allows to `drilldown` its value. This behavior is not suitable for `Select` in `multiple` mode.
- Extending `Dictionary`'s implementation leads to increasing complexity of  its testing and overriding, I think.

Reply:
@Dergash 
I've got it. I'll replace it by refactored `Dictionary`.
But I think its usage will look like:
```
case FieldType.multipleSelect:
    resultField = <Dictionary
        {...commonProps}
        multiple
        onChange={handleChange}
    />
```
Ok?